src/pages/faq.tsx

```diff
@@ -0,0 +1,64 @@
+import { Collapse, Text } from '@geist-ui/core';
+import { useRouter } from 'next/router';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+
+const FAQ = () => {
+  const FAQ = [
+    {
+      Name: 'AvdanOS',
+      Questions: [
+        {
+          QuestionNumber: 1,
```

Comment:
What exactly is QuestionNumber for? Is it possible to use index when looping through the array or use .length property?

Reply:
Its for a feature where you can go to first question by `avdanos.com/faq/AvdanOS#1` on the faq page

Reply:
Ohh, but i will change it

Reply:
And i removed it, i need to use index but then i read somewhere that using index as key is not good it causes side effects

Reply:
But okay, now deleted it